dev-kubernetes-manifests/userservice.yaml

```diff
@@ -80,8 +80,8 @@ spec:
           timeoutSeconds: 10
         resources:
           requests:
-            cpu: 200m
-            memory: 64Mi
+            cpu: 300m
+            memory: 128Mi
```

Comment:
@bourgeoisor first off, thank you thoroughly testing this pull-request.

**Question:** Have I interpreted [your suggestions](https://github.com/GoogleCloudPlatform/bank-of-anthos/pull/1054#issuecomment-1292450452) correctly?

Reply:
Yep, thank you very much, Nim!